client/state/plans/selectors.js

```diff
@@ -53,3 +53,14 @@ export function getPlanRawPrice( state, productId, isMonthly = false ) {
 	}
 	return isMonthly ? parseFloat( ( plan.raw_price / 12 ).toFixed( 2 ) ) : plan.raw_price;
 }
+
+/**
+ * Returns the user currency code
+ * @param  {Object}  state     global state
+ * @param  {Number}  productId the plan productId
+ * @return {String}  currency code, ie 'USD'
+ */
+export function getCurrencyCode( state, productId ) {
+	const plan = getPlan( state, productId );
```

Comment:
In this case getPlan can handle the null productId, and below we can handle a falsey plan. There isn't much of a consensus, but since we often wait for information to come back on a request, hardening our selectors to be able to handle falsey values is useful.